src/pages/docs/octopus-rest-api/openid-connect/index.md

```diff
@@ -233,3 +233,7 @@ Public sites such as [jwt.io](https://jwt.io/) can be used to inspect and valida
 
 IMPORTANT: Access tokens are credentials to your Octopus Server in the same way that API keys are, be careful where you paste them!
 :::
+
+## Older Versions
+
+- The `login` command was added to the Octopus CLI in version `2.1.0`.
```

Comment:
I think it would make it clearer and more helpful to users to add something like "Prior to 2.1.0, the CLI did not support OpenID connect".

Reply:
Good call, done